src/Components/BookItem.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const BookItem = (props) => {
+  const { bookProps } = props;
+  return (
+    <>
+      <p>{bookProps.title}</p>
+      <p>{bookProps.author}</p>
+      <button type="button">Delete</button>
+    </>
+  );
+};
+
+BookItem.propTypes = {
+  // eslint-disable-next-line react/forbid-prop-types
```

Comment:
- [ ] [OPTIONAL] Kindly set up the PropTypes correctly and try to not ignore the linters rules, they are in place to make your code clean

Reply:
Thank you @Ismailco for your time and review, I will fix your comments.